WireAPI/Tests/WireAPITests/APIs/BackendInfoAPI/BackendInfoAPITests.swift

```diff
@@ -16,47 +16,40 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-@testable import WireAPI
+import SnapshotTesting
 import XCTest
 
+@testable import WireAPI
+
 final class BackendInfoAPITests: XCTestCase {
 
-    private var httpClient: HTTPClient!
+    private let snapshotter = HTTPRequestSnapshotHelper()
```

Comment:
Hint: due to a not intuitive memory handling by Xcode properties like this will stay in memory until all test cases finished execution. Therefore usually it's recommended to create properties in `setUp` and nullify in `tearDown`.

From my point of view now in this case keep it as it is.